tests/integration/placements/test_placements.py

```diff
@@ -68,50 +68,40 @@ def test_placement_group_view(create_placement_group):
     assert_headers_in_lines(headers, lines)
 
 
-@pytest.mark.skip(reason="BUG TPT-3109")
 def test_assign_placement_group(support_test_linode_id, create_placement_group):
     linode_id = support_test_linode_id
     placement_group_id = create_placement_group
-    res = (
-        exec_test_command(
-            BASE_CMD
-            + [
-                "assign-linode",
-                placement_group_id,
-                "--linodes",
-                linode_id,
-                "--text",
-                "--delimiter=,",
-            ]
-        )
-        .stdout.decode()
-        .rstrip()
+    process = exec_test_command(
+        BASE_CMD
+        + [
+            "assign-linode",
+            placement_group_id,
+            "--linode",
+            linode_id,
+            "--text",
+            "--delimiter=,",
+        ]
     )
-    assert placement_group_id in res
+    assert process.returncode == 0
```

Comment:
Fyi, returncode assertion is done in exec_command function. Can we have different set of assertions that linode actually got assigned?